docs/docs/Tutorials/chat-with-files.md

```diff
@@ -0,0 +1,173 @@
+    * `FLOW_ID`: The UUID of your flow, or the endpoint name you've chosen.
+    * File Component name: To find the ID of your file component, in the File component, click **Controls**. For this example, the component name is `File-KZP68`.
+    * Input value: This is the message you want to send to the Chat Input of your flow, such as `Evaluate this resume for a job opening in my Marketing department.`
+    * File path: The path to the local file you want to load with your request. This example is loading `fake-resume.txt` from the same file location as the script.
+    * Session ID: Optional. For more information, see [session ID](/session-id).
+    * Langflow API key: Required. To create an API key, see [API keys](/configuration-api-keys).
+
+2. Replace the values in the script below.
+
+    <details open>
+        <summary>Python</summary>
+
+    ```python
+    # Python example using requests
+    import requests
+    import json
+
+    # 1. Set the upload URL
+    url = "http://LANGFLOW_SERVER_ADDRESS/api/v2/files/"
+
+    # 2. Prepare the file and payload
+    payload = {}
+    files = [
+      ('file', ('fake-resume.txt', open('fake-resume.txt', 'rb'), 'application/octet-stream'))
+    ]
+    headers = {
+      'Accept': 'application/json',
+      'x-api-key': 'LANGFLOW_API_KEY'
+    }
+
+    # 3. Upload the file to Langflow
+    response = requests.request("POST", url, headers=headers, data=payload, files=files)
+    print(response.text)
+
+    # 4. Get the uploaded file path from the response
+    uploaded_data = response.json()
+    uploaded_path = uploaded_data.get('path')
+
+    # 5. Call the Langflow run endpoint with the uploaded file path
+    run_url = "http://LANGFLOW_SERVER_ADDRESS/api/v1/run/FLOW_ID"
+    run_payload = {
+        "input_value": "Evaluate this resume for a job opening in my Marketing department.",
+        "output_type": "chat",
+        "input_type": "chat",
+        "tweaks": {
+            "File-KZP68": {
+                "path": uploaded_path
+            }
+        }
+    }
+    run_headers = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+        'x-api-key': 'LANGFLOW_API_KEY'
+    }
+    run_response = requests.post(run_url, headers=run_headers, data=json.dumps(run_payload))
+    langflow_data = run_response.json()
+    # Output only the message
+    message = None
+    try:
+        message = langflow_data['outputs'][0]['outputs'][0]['results']['message']['data']['text']
+    except (KeyError, IndexError, TypeError):
+        pass
+    print(message)
+
+    ```
+    </details>
+
+    This script contains two requests.
+
+    The first request uploads `fake-resume.txt` to your Langflow server at the `/api/v2/files` endpoint, which returns a file path that can be referenced in subsequent Langflow requests, `02791d46-812f-4988-ab1c-7c430214f8d5/fake-resume.txt`.
+
+    The second request sends a chat message to the Langflow flow at the `/api/v1/run/` endpoint.
```

Comment:
```suggestion
    The second request sends a chat message to the Langflow flow at the `/v1/run/` endpoint.
```

Another thing we should standardize on is how much of the path to include for these... should we always include /v1 or /v2 even if it only has one? and should we include /api/ or not?